src/App.js

```diff
@@ -1,8 +1,25 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './App.scss';
 
-const App = () => (
-  <h1>React sum</h1>
-);
+function App({ a, b }) {
+  return (
+    <p>
+      Sum of
+      {a}
+      {' '}
+      and
+      {b}
+      {' '}
+      is
+      {a + b}
```

Comment:
Here is the problem why your paragraph has no spaces between numbers. Fix it, and don't use the command `npm run lint`.

Reply:
I fixed this problem. But npm run lint use automatically when you make commit